diff -pur USD/cmake/modules/FindHoudini.cmake USD-mod/cmake/modules/FindHoudini.cmake
--- USD/cmake/modules/FindHoudini.cmake	2019-06-10 11:49:42.000000000 -0600
+++ USD-mod/cmake/modules/FindHoudini.cmake	2019-06-10 11:50:42.000000000 -0600
@@ -69,6 +69,8 @@ find_path(HOUDINI_BASE_DIR
         "$ENV{HOUDINI_ROOT}"
     )
 
+message("HOUDINI_BASE_DIR="${HOUDINI_BASE_DIR})
+
 find_path(HOUDINI_INCLUDE_DIRS 
     UT/UT_Version.h
     HINTS
@@ -76,19 +78,23 @@ find_path(HOUDINI_INCLUDE_DIRS 
         "$ENV{HOUDINI_ROOT}"        
         "${HOUDINI_BASE_DIR}"
     PATH_SUFFIXES
-        toolkit/include/
+        Resources/toolkit/include/
     DOC
         "Houdini Development Kit Header Path"
 )
 
 if (UNIX)
-    set(HOUDINI_LIB_NAME "libHoudiniGEO.so")
-    set(HOUDINI_LIB_PATH_SUFFIX "dsolib/")
+    set(HOUDINI_LIB_NAME "libHoudiniGEO${CMAKE_SHARED_LIBRARY_SUFFIX}")
+    if (APPLE)
+        set(HOUDINI_LIB_PATH_SUFFIX "Libraries/")
+    endif()
 elseif(WIN32)
     set(HOUDINI_LIB_NAME "libGEO.lib")
     set(HOUDINI_LIB_PATH_SUFFIX "custom/houdini/dsolib/")
 endif()
 
+message("HOUDINI_LIB_NAME="${HOUDINI_LIB_NAME})
+
 find_path(HOUDINI_LIB_DIRS 
     ${HOUDINI_LIB_NAME}
     HINTS
